Source/Core/DolphinWX/Input/HotkeyInputConfigDiag.cpp

```diff
@@ -68,6 +69,31 @@ HotkeyInputConfigDialog::HotkeyInputConfigDialog(wxWindow* const parent, InputCo
   // Frame advance is an example of a typical TAS tool.
   notebook->AddPage(tab_tas, _("TAS Tools"));
 
+  if (withDebugging)
+  {
+    // Debugging
+    auto* const tab_debugging = new wxPanel(notebook);
```

Comment:
oh okay,

I actually would prefer that, btw, the wiimote one would need the same separation too.